tools/changelog.main.kts

```diff
@@ -182,14 +182,14 @@ fun generateChangelog() {
 
             entries
```

Comment:
Should merge only after 1.8.0 Stable, as we still need to handle the previous PR without Release Notes